airflow/config_templates/config.yml

```diff
@@ -500,6 +500,13 @@ core:
       type: integer
       example: ~
       default: "4096"
+    disalble_scan_stale_dags:
+      description: |
+        Stale dags are deleted by default, [core] disalble_scan_stale_dags is True if you want to keep them.
```

Comment:
Correct this in context of above comment

Reply:
fixed